src/pal/src/locale/unicode.cpp

```diff
@@ -159,6 +159,22 @@ BOOL GetUnicodeData(INT nUnicodeValue, UnicodeDataRec *pDataRec)
     }
     return bRet;
 }
+
+/*++
+ -Function :
+ -CODEPAGEInit -
+ -
+ -    Initialize the LC_MESSAGES locale to en_US for the purpose of
+ -    the native resources string
+ ---*/
+void CODEPAGEInit( void )
+{
```

Comment:
This method has nothing to do with codepages now. Maybe it would be best to call this in PAL_BindResources? (Or rename the method and move it next to PAL_BindResources.)